README.md

```diff
@@ -58,6 +58,35 @@ This is currently the roadmap, please feel free to request additions/changes.
 
 PRs Welcome!
 
+## Setting up dev
+
+Easiest way to setup your development environment is simply opening the project in VScode (Dart and Flutter extensions too). You will also need to download Flutter's SDK and add it to your PATH, you can do so in the following [link](https://flutter.dev/docs/get-started/install)
+.
+
+ After the installation process, you will need to set up a device or emulator; Follow the instructions in the following links depending on your choose.
+
+- [iOS](https://flutter.dev/docs/get-started/install)
+- [android](https://flutter.dev/docs/get-started/install/macos#android-setup)
+
+If you already have an emulator or device recogniced by VSCode, then you just need to launch the app by going to `main.dart` and pressing `F5`.
+
+### Troubleshouting 
+
+- Running `flutter doctor` might hint you some issues with missing dependencies. Such as:
+
+```
+[✗] Android toolchain - develop for Android devices
+    ✗ Unable to locate Android SDK.
+      Install Android Studio from: https://developer.android.com/studio/index.html
+      On first launch it will assist you in installing the Android SDK components.
+      (or visit https://flutter.dev/docs/get-started/install/macos#android-setup for detailed instructions).
+      If the Android SDK has been installed to a custom location, please use
+      `flutter config --android-sdk` to update to that location.
+```
+
+- If running `flutter create .` fails; check the folder project name and make it compliant with Dart [rules](https://dart.dev/tools/pub/pubspec#name) .
```

Comment:
I now know why it initially failed. The problem is that the project until now was only used for Android. So if you want to run it on an iOS device / simulator, it is required to enable the current project for iOS. This will be done by this `flutter create .`command.